pkg/api/v2/types.go

```diff
@@ -376,6 +376,13 @@ const (
 
 	// DELETING state indicates the cluster is being deleted.
 	DELETING ExternalClusterState = "DELETING"
+
+	// UNSPECIFIED Not set.
+	UNSPECIFIED ExternalClusterState = "UNSPECIFIED"
```

Comment:
In other places we use "unknown". What do you think about aligning it?

Reply:
sounds good, I will change

Reply:
done